crypto/evp/digest.c

```diff
@@ -680,7 +681,7 @@ const OSSL_PARAM *EVP_MD_CTX_settable_params(EVP_MD_CTX *ctx)
 
     if (ctx->digest != NULL && ctx->digest->settable_ctx_params != NULL)
         return ctx->digest->settable_ctx_params(
-                  ossl_provider_ctx(EVP_MD_provider(ctx->digest)));
+                ctx->provctx, ossl_provider_ctx(EVP_MD_provider(ctx->digest)));
```

Comment:
This indent is pretty horrible.

Reply:
What do you suggest?  It's too long...

Reply:
Maybe

````
{
   blarg *b = a;
   return fgggjkkgdkgd(b);
}
````

Reply:
@slontis made a good suggestion, please do that

Reply:
There are a few more of these indents in this PR.

Reply:
Yes, I hope it's understood to be a broad recommendation and not just for this spot.

Reply:
They are all pre-existing indentation 😞

Reply:
That sucks, of course...  should have been caught earlier.

Reply:
More than likely I probably did it :)

Reply:
Fixed now.